test/test-create-options.js

```diff
@@ -12,13 +12,16 @@ test('create-options:', function (t) {
     module: {envVar: {testenvVar: 'thisisatest'}}
   };
 
+  //ensure that there is no environment variable defined
+  delete process.env['npm_config_tmp'];
```

Comment:
I would prefer this to be set to null rather than delete. /cc @jasnell for an opinion